rtp/Cargo.toml

```diff
@@ -13,6 +13,7 @@ once_cell = "1"
 # instead of relying on multiple TLS libraries.
 openssl = { version = "0.10", features = ["vendored"] }
 openssl-sys = "^0.9"
+serde = "1"
```

Comment:
Maybe a comment that we can move this dep if we move the id generation to a util crate?